etc/testing/vault-s3-client/test.sh

```diff
@@ -5,7 +5,11 @@ SCRIPT_DIR="$(dirname "${0}")"
 set -ex
 
 # Build pachyderm and push the relevant images
-make install && make docker-build && make docker-push || exit 1
+make install || exit 1
+VERSION=$(pachctl version --client-only)
+git tag -f -am "Travis test v${VERSION}" v"${VERSION}" || exit 1
+make docker-build || exit 1
+make docker-push || exit 1
```

Comment:
aren't all these `|| exit 1`s not doing anything, given that `-e` is set?

IOW, doing "or fail" on something that will fail if it fails anyway is the same as doing nothing?